doc/man/8/radosgw-admin.rst

```diff
@@ -455,20 +814,16 @@ Quota Options
 
 .. option:: --max-size
 
-	Specify max size (in bytes, negative value to disable).
+	Specify max size (negative value to disable).
```

Comment:
how about: Specify max size in bytes

Reply:
Please see https://github.com/joscollin/ceph/blob/a12ded9805024d334d9e625a8324463e8f1a4b88/src/rgw/rgw_admin.cc#L308. It is not just `in bytes`, but in other sizes too. That is the reason I removed `in bytes` here.

Reply:
can you use the same text as in the code?

Reply:
Done. Please verify.